### DISCUSSION TITLE
bootstrap 5 datalist more than 1 letter problem

### DISCUSSION THREAD

**Discussion by efeeker** [twbs/bootstrap#34274] [Q&A]
![1](https://user-images.githubusercontent.com/16708047/122283918-bdb50a80-cef5-11eb-82bf-482c9f5033d9.jpg)
![2](https://user-images.githubusercontent.com/16708047/122283930-bee63780-cef5-11eb-9dba-429e5e9ef718.jpg)

how i can fix this problem

**Comment by mdo** [comment]
You're interacting with a datalist in our docs it looks like. Unless you've modified the `<datalist>` values, I don't see how Istanbul would show up. There's no JS or anything that's controlling this—it's just HTML.

**Reply by efeeker** [reply]
> You're interacting with a datalist in our docs it looks like. Unless you've modified the `<datalist>` values, I don't see how Istanbul would show up. There's no JS or anything that's controlling this—it's just HTML.

thanks for answer. i was added in html, when i was add english letters no problem but when i was add turkish first letter is uppercase i " İ " doesn't work
